pkg/db/query.go

```diff
@@ -283,6 +285,24 @@ func (s *Select) GetMany() ([]DatabaseEntity, error) {
 	return result, nil
 }
 
+func (s *Select) sql() string {
```

Comment:
It would be nice to have a unit test for this function 👍🏻

Reply:
done